src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/innerassignment/InputInnerAssignment.java

```diff
@@ -33,6 +33,20 @@ void innerAssignments()
         }
     }
 
+    public void expressionCanHaveParentSwitchCaseIssue13086()
```

Comment:
Please do not extend existing inputs. We need to create a completely new test and test input using switch statements, switch expressions, with both switch rules and labeled block statements. You will need to create the new input in resources-noncompilable in order to support syntax greater than jdk11.

Reply:
Yes, please create new Input file, this input file is already too big.

Reply:
Please in addition add test case from user, to be absolutely clear that user reported case is covered

Reply:
@nrmancuso @romani  This issue also exists when I ran with Java 11, so should I simple create a new test file in `src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/innerassignment ` instead of adding it in resources-noncompilable

Reply:
No, we must include exact case from issue report.

Reply:
@Andromeda227799 always allow reviewers to resolve their items, please just reply to each with “done” or discussion if necessary.

Reply:
Sure, understood

Reply:
Done